### DISCUSSION TITLE
ventoy with Finnix

### DISCUSSION THREAD

**Discussion by datamaskinen** [finnix/finnix#32] [General]
Hi,
I have just started to use https://www.ventoy.net because it is so easy to just throw on some isos and its done way to get my backup restore and doctor kit for servers.... and the finnix does not seem to work with it . is there something  that i should ask ventoy about or finnix? i tried to contact ventoy but have not got any reply so i am asking finnix as well then ...

every usb boot stick should have a finnix option on it :)

**Comment by CornelisJ** [comment]
FYI, I just booted my PC with the finnix-124.iso from a USB stick with Ventoy 1.072 installed on it, and Finnix is booting and working without problem. So it is working for me. This was with a PC with UEFI bios.

**Reply by datamaskinen** [reply]
hmmm ok it might be my bios or something that is not supported then ... thanx a lot for trying it out ... 

Best regards
-mL-

**Reply by Ixw123** [reply]
What errors, if any were there?
